R/module_init_data.R

```diff
@@ -128,7 +128,7 @@ srv_init_data <- function(id, data) {
     function(dataname, datasets) {
       hash <- rlang::hash(data[[dataname]])
       sprintf(
-        "stopifnot(%s == %s) # @linksto %s",
+        "stopifnot(%s == %s) #@linksto %s",
```

Comment:
This is always inside strings, however the styler ruleset that we have in insightengineering enforces a space after the `#`

Would it be better to change the other instance?

Reply:
No strong opinion as it is inside a string. Without the space there is less characters :D

Reply:
I'm having second thoughts and I would actually write # comment (hasthag space somment) when I would type a comment, so let's go with your approach

Reply:
Sounds good either way.